### PR TITLE
Use php7.0

### PR DESCRIPTION
**Pull request by goruha**
## What
* Use php 7.0 fpm
* Enabled travis ci 
* Refactor dir structure

## Why
* Need php 7.0 for one project
* To build containers the same way
* To follow on pattern for containers repo structure

### CODE REVIEW

**Comment by const-bon** [comment]
~Build failed.~
`docker build -t cloudposse/apache-php-fpm .`
image

**Comment by const-bon** [comment]
Above error happens on `master` branch.

File: rootfs/etc/php/7.0/fpm/php-fpm.conf [hunk 1]
```diff
@@ -87,18 +79,18 @@ error_log = /var/log/php5-fpm.log
...
- 
+
 ; Set max core size rlimit for the master process.
 ; Possible Values: 'unlimited' or an integer greater or equal to 0
 ; Default Value: system defined value
 ;rlimit_core = 0
 
 ; Specify the event mechanism FPM will use. The following is available:
-; - select     (any POSIX os)
+  ; - select     (any POSIX os)
```
Comment (const-bon) [inline rootfs/etc/php/7.0/fpm/php-fpm.conf:93]: Drop unnecessary white spaces.

**Comment by const-bon** [comment]
Looks ok.
`$ docker build -t cloudposse/apache-php-fpm .`
image

Inside the container
```
# echo "<?php phpinfo(); ?>" >/var/www/html/index.php
# curl localhost
```
image

**Review by const-bon — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -0,0 +1 @@
+-include $(shell curl -sSL -o .build-harness "https://git.io/build-harness"; echo .build-harness)
```
Comment (const-bon) [inline Makefile:1]: Add newline at the end of each file.

**Review by osterman — APPROVED** [review]

**Pull request merged**